src/pages/chatBotScreen/chatBotScreen.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I am not sure about the name chatBotScreen if you can think of something else. You can call it just chatbot

And also Change the name of the components to start with a capital letter like: `ChatBot`